kinto/plugins/history/views.py

```diff
@@ -57,3 +61,85 @@ def _extract_filters(self):
             filters_str_id.append(filt)
 
         return filters_str_id
+
+
+version_view = Service(name='version_view',
+                       description='Handle retrieving object from the past.',
+                       path='{subpath:.*}/version/{last_modified:[0-9]{13}}')
+
+
+@version_view.get(permission=NO_PERMISSION_REQUIRED)
+def get_version_view(request):
+    last_modified = int(request.matchdict['last_modified'])
+    subpath = '/{}'.format(request.matchdict['subpath'])
+
+    if not subpath.startswith('/buckets'):
+        raise httpexceptions.HTTPNotFound()
+
+    bucket_id = request.matchdict['subpath'].split('/', 2)[1]
```

Comment:
Why all of this instead of putting `/buckets/{bucket_id}/{subpath:.*}` in the path?